ci-operator/step-registry/upi/deprovision/heterogeneous/vpc/upi-deprovision-heterogeneous-vpc-commands.sh

```diff
@@ -59,38 +54,15 @@ function retry {
   done
 }
 
-function setup_jq() {
-  if [ -z "$(command -v jq)" ]
-  then
-    echo "jq is not installed, proceed to installing jq"
-    curl -L "https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64" -o /tmp/jq && chmod +x /tmp/jq
-  fi
-}
-
-function setup_ibmcloud_cli() {
-  if [ -z "$(command -v ibmcloud)" ]
-  then
-    echo "ibmcloud CLI doesn't exist, installing"
-    curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
-  fi
-
-  retry "ic config --check-version=false"
-  retry "ic version"
-}
-
 function setup_terraform_cli() {
-  if [ -z "$(command -v terraform)" ]
-  then
-    echo "terraform CLI doesn't exist, installing"
-  fi
```

Comment:
We always want to pickup terraform (the version in the container is v1.0.0 or so, not v1.5.5) so the if wasn't providing anything useful.

Reply:
Pinged comments directly